packages/components/AssetsPagination.tsx

```diff
@@ -0,0 +1,177 @@
+import React from "react";
+import { StyleSheet, TextInput, TouchableOpacity, View } from "react-native";
+
+import { BrandText } from "./BrandText";
+import { SVG } from "./SVG";
+import { SEARCH_BAR_INPUT_HEIGHT } from "./Search/SearchBarInput";
+import { Box } from "./boxes/Box";
+import { LegacyTertiaryBox } from "./boxes/LegacyTertiaryBox";
+import { TertiaryBox } from "./boxes/TertiaryBox";
+import { SpacerRow } from "./spacer";
+import chevronLeftDoubleSVG from "../../assets/icons/chevron-left-double.svg";
+import chevronLeftSVG from "../../assets/icons/chevron-left.svg";
+import chevronRightDoubleSVG from "../../assets/icons/chevron-right-double.svg";
+import chevronRightSVG from "../../assets/icons/chevron-right.svg";
+import {
+  neutral17,
+  neutral33,
+  neutral77,
+  primaryColor,
+} from "../utils/style/colors";
+import { fontSemibold14 } from "../utils/style/fonts";
+import { layout } from "../utils/style/layout";
+
+interface PaginationProps {
+  currentPage: number;
+  maxPage: number;
+  itemsPerPage: number;
+  dropdownOptions: number[];
+  setItemsPerPage: (item: number) => void;
+  onChangePage: (page: number) => void;
+}
+
+export const AssetsPagination = ({
+  currentPage,
+  maxPage,
+  dropdownOptions,
+  itemsPerPage,
+  setItemsPerPage,
+  onChangePage,
+}: PaginationProps) => {
+  const handleChangePage = (pageIndex: number) => {
+    if (pageIndex < 0) {
+      pageIndex = 0;
+    } else if (pageIndex >= maxPage) {
+      pageIndex = maxPage - 1;
+    }
+    if (pageIndex !== currentPage) {
+      onChangePage(pageIndex);
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      <View style={{ flex: 1, flexDirection: "row" }}>
+        <View style={[styles.section, { justifyContent: "flex-start" }]}>
+          <BrandText style={styles.grayText}>
+            Page {currentPage + 1} of {maxPage}
+          </BrandText>
+        </View>
+        <View style={[styles.section, { justifyContent: "flex-start" }]}>
+          <BrandText style={styles.grayText}>| Go to page:</BrandText>
+          <TertiaryBox
+            style={[
+              {
+                flexDirection: "row",
+                paddingHorizontal: 12,
+                backgroundColor: neutral17,
+                width: 80,
+                height: SEARCH_BAR_INPUT_HEIGHT,
+                alignItems: "center",
+              },
+            ]}
+          >
+            <TextInput
+              value={currentPage.toString()}
+              inputMode="numeric"
+              style={[
+                fontSemibold14,
+                {
+                  color: "white",
+                  flex: 1,
+                  width: 56,
+                },
+                { outlineStyle: "none" } as any,
+              ]}
+              onChangeText={(text) => {
+                handleChangePage(Number(text));
```

Comment:
Just FYI: you can use +text to quick convert to number

Reply:
Okay, I will fix it. Thanks